editor/src/clj/editor/sprite.clj

```diff
@@ -467,8 +474,7 @@
                        :error (validation/prop-error :info _node-id :texture validation/prop-nil? nil label)
                        :edit-type {:type resource/Resource
                                    :ext extension
-                                   :set-fn (fn [_ _ _ new]
-                                             (create-texture-binding-tx _node-id sampler-name new))}}])))))
+                                   :set-fn (fxui/partial set-texture-biding-id sampler-name)}}])))))
```

Comment:
Perhaps we should consider moving `fxui/partial` to the `util.fn` module? It seems generally useful!